layouts/_default/article.html

```diff
@@ -14,7 +14,7 @@
             <h3 class="text-xl font-extrabold mt-3 pb-2 "><a class="block h-24" href="{{ .Permalink }}">{{ .Title }}</a>
             </h3>
             <div class='art-s'>
-                <p class="mt-3 text-base font-extralight">{{ .Summary | safeHTML | truncate 240 }}</p>
+                <p class="mt-8 text-base font-extralight">{{ .Summary | safeHTML | truncate 240 }}</p>
```

Comment:
The length of the blog post's title required we add a little more space between the title and description so that it will look fine on smaller screens.